src/components/pengestopp/PengestoppModal.tsx

```diff
@@ -177,56 +160,57 @@ const PengestoppModal = ({
     setEmployerError(false);
     setAarsakError(false);
 
-    toggle();
+    onModalClose();
   };
 
   return (
-    <StyledModal
+    <Modal
+      closeOnBackdropClick
+      className="p-8 max-w-4xl w-full"
       aria-label={texts.stansSykepenger}
       open={isOpen}
       onClose={handleCloseModal}
     >
-      <Modal.Content>
-        <Systemtittel>{tittel(isSuccess)}</Systemtittel>
-
+      <Modal.Header>
+        <Heading size="medium">{tittel(isSuccess)}</Heading>
+      </Modal.Header>
+      <Modal.Body>
         {!isSuccess ? (
           <>
-            <Group>
-              <CheckboxGruppe
-                legend={texts.arbeidsgiver}
-                feil={employerError && texts.submitError}
-              >
-                {arbeidsgivere.map(
-                  (arbeidsgiver: Arbeidsgiver, index: number) => {
-                    return (
-                      <Checkbox
-                        key={index}
-                        label={arbeidsgiver.navn}
-                        onChange={handleChangeVirksomhet}
-                        name={arbeidsgiver.orgnummer}
-                      />
-                    );
-                  }
-                )}
-              </CheckboxGruppe>
-            </Group>
-            <Group>
-              <CheckboxGruppe
-                legend={texts.arsak.title}
-                feil={aarsakError && texts.arsak.submitError}
-              >
-                {sykepengestoppArsakTekstListe.map((arsak, index: number) => {
+            <CheckboxGruppe
+              className="my-4"
+              legend={texts.arbeidsgiver}
+              feil={employerError && texts.submitError}
+            >
+              {arbeidsgivere.map(
+                (arbeidsgiver: Arbeidsgiver, index: number) => {
                   return (
                     <Checkbox
                       key={index}
-                      label={arsak.text}
-                      onChange={handleChangeArsak}
-                      name={arsak.type}
+                      label={arbeidsgiver.navn}
+                      onChange={handleChangeVirksomhet}
+                      name={arbeidsgiver.orgnummer}
```

Comment:
Lurer på om det heller burde vært `value={arbeidsgiver.orgnummer}` her? 😊 `name` attributen brukes som oftest til å identifisere grupperte elementer, eller navngi felter.
Hvis `name` skal ha en verdi funker kanskje `"arbeidsgiver"`? 😁
```suggestion
    name={"arbeidsgiver"} 
```

Reply:
Hmja, det stemmer kanskje det. Har ikke endret noe på denne koden her annet enn innrykk fordi jeg fjernet en wrapper-komponent, så tenker jeg lar det være sånn det er i denne omgang.